deepchem/models/torch_models/chemberta.py

```diff
@@ -60,12 +60,12 @@ class Chemberta(HuggingFaceModel):
     >>> from deepchem.models.torch_models.chemberta import Chemberta
     >>> pretrain_model_dir = os.path.join(tempdir, 'pretrain-model')
     >>> tokenizer_path = "seyonec/PubChem10M_SMILES_BPE_60k"
-    >>> pretrain_model = Chemberta(task='pretraining', model_dir=pretrain_model_dir, tokenizer_path=tokenizer_path, mode='mlm')
+    >>> pretrain_model = Chemberta(task='pretraining', model_dir=pretrain_model_dir, tokenizer_path=tokenizer_path)
```

Comment:
MLM and MTR are two different pretraining tasks though. How do we differentiate between MLM pretraining and MTR pretraining?